travis/publish.sh

```diff
@@ -85,16 +85,16 @@ release_version() {
     echo "${TRAVIS_TAG}" | sed 's/^release-//'
 }
 
-safe_checkout_master() {
+safe_checkout_remote_branch() {
   # We need to be on a branch for release:perform to be able to create commits, and we want that branch to be master.
```

Comment:
Should the references to 'master' in this comment block also be changed?

Reply:
I have just pushed a comment change